test/TestGrains/ProgrammaticSubscribe/SubscribeGrain.cs

```diff
@@ -64,10 +64,10 @@ public Task<IEnumerable<StreamSubscription>> GetSubscriptions(FullStreamIdentity
             return subManager.GetSubscriptions(streamIdentity.ProviderName, streamIdentity);
         }
 
-        public async Task RemoveSubscription(StreamSubscription subscription)
+        public async Task RemoveSubscription(FullStreamIdentity streamId, Guid subscriptionId)
```

Comment:
there's tests testing `SubscrptionManager.RemoveSubscription` before, it is just not using custom implemented `IStreamIdentity`, this change would make that test testing this scenario.

Reply:
because `SubscrptionManager.RemoveSubscription` is using `FullStreamIdentity ` which is a custom implementation for `IStreamIdentity`, directly.